src/odoc/resolver.mli

```diff
@@ -21,10 +21,17 @@
 
 type t
 
+type roots = {
+  page_pkgnames : (string * Fs.Directory.t) list;
+  lib_pkgnames : (string * Fs.Directory.t) list;
+  current_pkg : string;
+}
+
 val create :
   important_digests:bool ->
   directories:Fs.Directory.t list ->
   open_modules:string list ->
+  roots:roots option ->
```

Comment:
I think docs should be updated to explain the roots options here

Reply:
Rereading this part, I noticed that some of my terminology might be wrong: for the `-P` it is the package name that is relevant, but for the `-L` it is the library name.